charts/newrelic-infrastructure/Chart.yaml

```diff
@@ -8,31 +8,33 @@ sources:
   - https://github.com/newrelic/nri-kubernetes/tree/main/charts/newrelic-infrastructure
   - https://github.com/newrelic/infrastructure-agent/
 
-version: 3.12.0
-appVersion: 3.6.0
+version: 3.13.0
+appVersion: 3.7.0
```

Comment:
We just released a new app image, so we'll use your PR to bump the appVersion as well. :)